test/features/pages/signin.js

```diff
@@ -0,0 +1,58 @@
+/* jshint node: true */
+
+'use strict';
+
+/*
+ * signin.js
+ *
+ * Sign in page object
+ * Detaches UI interactions from step definitions
+ *
+ */
+
+var testUsername = 'test_prison_1';
+var testPassword = 'test_prison_1';
+
+var SigninPage = function () {
+  this.usernameField = '#id_username';
+  this.passwordField = '#id_password';
+  this.signinForm = '#login-form';
+
+  this.get = function (callback) {
+    browser
+      .url('/auth/login/')
+      .call(callback);
+  };
+
+  this.fillForm = function (username, password, callback) {
+    browser
+      .setValue(this.usernameField, username)
+      .setValue(this.passwordField, password)
+      .submitForm(this.signinForm)
+      .call(callback);
+  };
+
+  this.signin = function (callback) {
+    var that = this;
+
+    this.get(function () {
+      that.fillForm(testUsername, testPassword, callback);
+    });
+  };
+
+  this.signinWith = function (username, password, callback) {
+    this.fillForm(username, password, callback);
+  };
+
+  this.signout = function (callback) {
+    browser
+      .url('/auth/logout/')
+      .call(callback);
+  };
```

Comment:
I was planning to change to that, but annoyingly, because of the feature tour, in some cases you'd have to click dismiss first and then the sign out link.

Need to find a better way to do that process. Possibly login with a cookie set that will disable the feature tour for the purpose of this test?